README.md

```diff
@@ -120,6 +120,12 @@ cargo build --release --features windows
 cargo build --release --features unix
 ```
 
+To build SELinux specific features including `chcon` and `runcon`, run
+```
+cargo build --release --features unix,feat_selinux
+```
+on the system with `libselinux` and `libclang` installed. 
+
```

Comment:
To improve the readability I would put the command after the text. Something like:

To build SELinux-specific features, including chcon and runcon, ensure that libselinux and libclang are installed on your system. Then, run the following command:
```
cargo build --release --features unix,feat_selinux
```